### ISSUE TITLE
Failure to resolve dependencies in Dart 2.12.0

### ISSUE THREAD

**Issue by parnham** [angulardart/angular_components#496]
The latest stable Dart release breaks the build for all of our applications that rely on angular_components due to the dependency on observable. It seems observable was patched the other week but no release has been made as yet.

angular: v6.0.1
angular_components: v1.0.2

```
Resolving dependencies... (1.4s)
The current Dart SDK version is 2.12.0.

Because no versions of observable match >0.22.2 <0.23.0 and angular_components >=0.13.0 depends on observable ^0.22.2, angular_components >=0.13.0 requires observable 0.22.2.
And because observable 0.22.2 depends on dart_internal ^0.1.1, angular_components >=0.13.0 requires dart_internal ^0.1.1.
So, because dart_internal ^0.1.1 requires SDK version >=2.0.0-dev.12.0 <2.12.0 and cog depends on angular_components ^1.0.2, version solving failed.
```

**Comment by dukefirehawk** [comment]
Add the following line to pubspec.yaml

dependency_overrides:
  dart_internal: ^0.2.0

**Comment by kevmoo** [comment]
This is now fixed with `1.0.3`
